controllers/status-check-override.js

```diff
@@ -0,0 +1,73 @@
+var scOverride = require('status-check');
+
+delete scOverride['checkSingleLink'];
+
+scOverride.checkSingleLink = function (linksArray, index, callBack, outputArray, showProgressInConsole) {
+  var presentObject = this,
+    http = require('http'),
+    https = require('https'),
+    method = https,
+    alterFlag = 0,
+    originalLink = "";
+  if (linksArray[index].indexOf('https://') === -1 && linksArray[index].indexOf('http://') === -1) {
+    originalLink = linksArray[index];
+    linksArray[index] = "http://" + linksArray[index];
+    alterFlag = 1;
+  }
+  if (linksArray[index].indexOf('https') == -1) {
```

Comment:
You should probably use `===` (You're using `==` and `!=` a bit everywhere in this piece of code, would be great to replace all of them to their `===` equivalent.)